tests/tasks/setup_802_1x_server.yml

```diff
@@ -4,6 +4,17 @@
   package:
     name: hostapd
     state: present
+  when:
+    - ansible_distribution_major_version != '8' or
+      ansible_distribution != 'RedHat' or
+      ansible_distribution_version | float >= 8.6
+
+- name: Install hostapd via CentOS Stream
+  command: dnf -y install http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/hostapd-2.10-1.el8.x86_64.rpm
+  when:
+    - ansible_distribution_version | float < 8.6 and
+      ansible_distribution_major_version == '8'
+    - ansible_distribution == 'RedHat'
```

Comment:
Moving this before the package command will allow to remove the when statements for the package command. Also, a comment that explains that this can be removed once RHEL 8.5 is not tested anymore, would be great. The commit message should mention that hostapd was moved from EPEL to CentOS Stream and is therefore not available anymore on EPEL, which made the installation fail.

Reply:
I updated, thanks.